CMake/lrs_options.cmake

```diff
@@ -37,6 +37,7 @@ option(FORCE_WINUSB_UVC "Explicitly turn-on winusb_uvc (for win7) backend - depr
 option(ANDROID_USB_HOST_UVC "Build UVC backend for Android - deprecated, use FORCE_RSUSB_BACKEND instead" OFF)
 option(CHECK_FOR_UPDATES "Checks for versions updates" ON)
 option(BUILD_WITH_CPU_EXTENSIONS "Enable compiler optimizations using CPU extensions (such as AVX)" ON)
+set(UNIT-TESTS "" CACHE STRING "Parameters to pass to unit-test-config.py")
```

Comment:
You can have a flag with a `-`?
I like it (especially if it works) but all the rest of the flags are with `_`... we should probably align.
For the description, let's be a little more specific: `Command-line arguments to pass to unit-tests-config.py, e.g. '-t <tag> -r <regex>'`
For the name, maybe we should be more specific, too? Like `UNIT_TESTS_ARGS`? What do you think?

Reply:
Done